### PR TITLE
Fix windows wheel

### LINKED ISSUE #733: [BUG] the Windows wheel(s) should contain librsync.dll and scripts be called .py
## Bug summary

The wheel doesn't work under Windows without the library, and Windows doesn't recognize `rdiff-backup` as python script, it should be called `rdiff-backup.py`, I never noticed because I don't use the wheel (and apparently nobody does either).

### PR DESCRIPTION
**Pull request by ericzolf**
FIX: make Windows wheel runnable by adding rsync.dll and renaming scripts, closes #733, #602

Use entry_points instead of scripts in setup.py. @ikus060 you were right in #602, it rocks (but makes things a bit more complicated with PyInstaller).

The scripts have disappeared from the source code and moved to modules.
The modules run_delete and run_stats are in rdiff_backup because ultimately they should become actions and disappear.
The setup.py has become yet a little more ugly to copy the DLL to the right place, but it'll need anyway a bigger revamp at some point in time. Most important, it works.

### CODE REVIEW

File: src/rdiffbackup/run.py [hunk 1]
```diff
@@ -23,6 +23,18 @@
 from rdiff_backup import Globals, log
 from rdiffbackup import arguments, actions_mgr, actions
 
+try:
+    import msvcrt
+
+    msvcrt.setmode(0, os.O_BINARY)
```
Comment (ikus060) [inline src/rdiffbackup/run.py:29]: A comment would be helpful to understand why it's needed

File: setup.py [hunk 2]
```diff
@@ -240,10 +257,19 @@ def run(self):
...
+            'rdiff-backup-delete = rdiff_backup.run_delete:main',
+            'rdiff-backup-statistics = rdiff_backup.run_stats:main',
+        ]
+    },
     packages=["rdiff_backup", "rdiffbackup",
               "rdiffbackup.actions", "rdiffbackup.utils", "rdiffbackup.meta",
               "rdiffbackup.locations", "rdiffbackup.locations.map"],
     package_dir={"": "src"},  # tell distutils packages are under src
+    include_package_data=True,
+    package_data={"rdiff_backup": ["*.dll"]},
```
Comment (ikus060) [inline setup.py:272]: Would that include the dll file for Linux too !?

Comment (ericzolf) [reply setup.py:272]: No, it just wouldn't exist. And that would simply be ignored.

File: src/rdiffbackup/run.py (hunk 1 above)
Comment (ericzolf) [reply src/rdiffbackup/run.py]: To be honest, I took over the code as it stands, and never cared as it worked. But, good point, and clarified.

**Pull request merged**